web/src/dark_theme.ts

```diff
@@ -12,6 +13,7 @@ export function enable(): void {
         const ls = localstorage();
         ls.set("spectator-theme-preference", "dark");
         user_settings.color_scheme = settings_config.color_scheme_values.night.code;
+        realm_logo.render();
```

Comment:
I was wondering why we don't need `realm_logo.render();` call for logged-in users and it seems like we do, so we move the call inside these `enable/ disable` functions as well.

Reply:
Makes sense, thanks! Moved `realm_logo.render()` along with `message_lists.update_recipient_bar_background_color()` into `dark_theme` module.